backend/index.js

```diff
@@ -45,7 +45,7 @@ app.set('view engine', 'njk')
 setupNunjucks(app)
 setupPhaseName(app, config)
 
-app.use(helmet())
+app.use(helmet({ contentSecurityPolicy: false }))
```

Comment:
csp is pretty handy to stop unexpected 3rd part js script references being injected into a page. It might be worth doing something like this after app.use(helmet()) instead:

```
app.use(
  helmet.contentSecurityPolicy({
    useDefaults: true,
    directives: {
      "script-src": ["'self'", "<add external host urls we want to allow here>.com"],
    },
  })
);
```